server/pulp/server/db/manage.py

```diff
@@ -200,6 +234,9 @@ def main():
         return os.EX_DATAERR
     except models.MigrationRemovedError:
         return os.EX_SOFTWARE
+    except ServerSelectionTimeoutError:
+        _logger.info(_('Cannot connect to database, please validate that the database is up.'))
```

Comment:
s/up/online and accessible/

I suggest ^ to capture that the db needs to be reachable via the network.